docs/sql-migration-guide.md

```diff
@@ -64,6 +64,7 @@ license: |
 ## Upgrading from Spark SQL 3.5.1 to 3.5.2
 
 - Since 3.5.2, MySQL JDBC datasource will read TINYINT UNSIGNED as ShortType, while in 3.5.1, it was wrongly read as ByteType.
+- Since 3.5.2, the `base64` function will return a non-chunked string. To restore the previous behavior where base64 encoded strings that chunked into lines of at most 76 characters, set `spark.sql.legacy.chunkBase64String.enabled` to `true`.
```

Comment:
the behavior changed in spark 3.3.0, so "the previous behavior" might be ambiguous.

Reply:
Makes sense, I made a little change, thank you.